pyuploadcare/dj/conf.py

```diff
@@ -24,7 +24,7 @@
     conf.cdn_base = settings.UPLOADCARE['cdn_base']
 
 
-widget_version = settings.UPLOADCARE.get('widget_version', '2.10.2')
+widget_version = settings.UPLOADCARE.get('widget_version', '2.10.0')
```

Comment:
We agreed that splitting copy methods to local and remote  would be a major change, and this branch would go into master along with the other changes introduced in the future. So the version change is symbolic. Not sure how to go about pull request though.